scripts/build_database.sh

```diff
@@ -409,7 +410,7 @@ create_taxon_tables() {
 		-e 's/pathogroup/no rank/' -e 's/forma specialis/no rank/' \
 		-e 's/serotype/no rank/' -e 's/clade/no rank/' \
 		-e 's/isolate/no rank/' -e 's/infraclass/no rank/' \
-		-e 's/parvorder/no rank/' "$TEMP_DIR/$UNIPEPT_TEMP_CONSTANT/nodes.dmp"
+		-e 's/parvorder/no rank/' -e 's/no_rank/domain/' "$TEMP_DIR/$UNIPEPT_TEMP_CONSTANT/nodes.dmp"
```

Comment:
For clarity: Are the substitutions sequential? As in, will the last substitution not set all previous replacements to 'domain'?

Reply:
It doesn't matter, since NCBI uses `no_rank` (with an underscore) and we've always used `no rank` (with a space). I will, anyways, update the order to improve the readability here :)